app/js/dialog.js

```diff
@@ -0,0 +1,8 @@
+jQuery(function($) {
+  $('#privacyModal')
+    .modal({backdrop: 'static', keyboard: false})
```

Comment:
This line prevents you from dismissing the modal by clicking on the backdrop or by hitting the Esc key.

This forces you to click the "I understand button".

Reply:
Nice touch, great attention to detail there